roles/pulp_webserver/tasks/apache.yml

```diff
@@ -6,25 +6,50 @@
         name: "{{ pulp_webserver_apache_package }}"
         state: present
 
+    - name: Change unix socket to ip
+      set_fact:
+        pulp_api_bind: >-
+          {% if pulp_api_bind.startswith('unix') %}
+          {{ pulp_api_bind }}|http://{{ pulp_api_apache_url }}
+          {% else %}
+          http://{{ pulp_api_bind }}
+          {% endif %}
+
     - name: Install Apache vhost file
       template:
         src: pulp-vhost.conf.j2
         dest: "{{ pulp_webserver_apache_vhost_dir }}/pulp-vhost.conf"
       notify: reload apache
 
-    - name: Enable Apache vhost files and mod files
+    - name: Enable Apache mod files
+      apache2_module:
+        state: present
+        name: "{{ item }}"
+      loop:
+        - headers
+        - proxy
+        - proxy_http
+        - ssl
+      when: ansible_facts.os_family == "Debian"
+      notify: reload apache
```

Comment:
:100:

Reply:
Fixing debian support for httpd should be a separate changelog entry (and commit, and bug). We want to let users know that we fixed it.

I do not see a need for separate bugs for all the issues you fixed, but still.